### DISCUSSION TITLE
A Git Repository cannot be connected to more than 3 Projects

### DISCUSSION THREAD

**Discussion by faha1999** [anuraghazra/github-readme-stats#2416] [Q&A] [answered]
I'm trying to host [github-readme-stats](https://github.com/anuraghazra/github-readme-stats) in my own **vercel** instance.

I follow all the steps which are in the **readme.md**

But I'm facing an issue when I click the **Deploy** button. It says `A Git Repository cannot be connected to more than 3 Projects.`

What should I do now to host in my own **vercel** instance?

**Comment by bharat-ghadi** [comment]
## I am also facing the same issue while trying to deploy . 
### Please let me know if someone knows the solution.
![Screenshot 2023-01-16 162626](https://user-images.githubusercontent.com/74781591/212662136-e8e24c00-6cd6-410e-a0b3-07cc22365fff.png)

**Comment by Zo-Bro-23** [comment, marked as answer]
How are you trying to deploy? You must first fork the repo before deploying.

**Comment by faha1999** [comment]
First, I fork the repo and try to deploy it in the vercel then face this issue. Also, change the **maxDuration field to 10.**

**Reply by Zo-Bro-23** [reply]
Hmmm. It worked fine for me. You can use my deployment if you want (`https://github-readme-stats.zohan.tech`).

**Comment by bharat-ghadi** [comment]
## after deletion of vercel account and creating forked repo in vecel solved my issue.

I already did the same. I forked repo first-hand, afterwards changed maxDuration field to 10.
But it wasn't working then. I deleted my vercel account completely and did same procedure again. It is working fine now. Anyways thanks for the prompt reply guys. Really appreciated. There was 3 vercel repo project created , maybe that was conflicting my deployment probably, Not sure.

**Reply by faha1999** [reply]
Can you tell me The details procedure? On the other side, I've a lot of repos in vercel. So, I can't completely delete the vecel account.

**Reply by bharat-ghadi** [reply]
### step 1: fork the repo into your github profile
### step 2: change vercel.json file in your forked repo
maxDuration: 30 before change it to 10
![Capture1](https://user-images.githubusercontent.com/74781591/212700180-8570bbd3-9521-4b3b-948a-f600f4c679ef.PNG)

### step 3: Next thing I did is I deleted my vercel account and created it again,  I might have created redundant project  so i decided to delete my ac.
![Capture3](https://user-images.githubusercontent.com/74781591/212700735-794e4c0f-aa52-4aae-b075-86b7a011925d.PNG)

### step 4: https://www.youtube.com/watch?v=n6d4KHSKqGk&t=228s
after all these steps follow this tutorial.
original repo link also mentioned into his YouTube video.

**Reply by bharat-ghadi** [reply]
You may try deleting relevant projects too, deleting vercel account seems extreme. Yet I didn't have any other projects in vercel so I opted for it.

**Reply by faha1999** [reply]
Thanks for your details reply. I follow all steps from the last 2 days and also watched this video too. But still, now I can't delete vercel account. So, I must have to use someone's deployment link. 

I hope your steps list will help others.

**Reply by Zo-Bro-23** [reply]
> Thanks for your details reply. I follow all steps from the last 2 days and also watched this video too. But still, now I can't delete vercel account. So, I must have to use someone's deployment link.
> 
> I hope your steps list will help others.

Feel free to use mine (`https://github-readme-stats.zohan.tech`)!

**Comment by Zo-Bro-23** [comment]
![image](https://user-images.githubusercontent.com/62234360/212698620-eef90683-128f-4571-80db-ecf047fa586b.png)

The error occurs when you try to make four projects with the same Git repo (fork). Maybe you got the `maxDuration` error, so you had to create multiple projects? See how many projects you have on Vercel, and see if any of those are linked to the repo you are trying to deploy.

**Reply by faha1999** [reply]
I already changed the **maxDuration** below 10
